tests/components/zwave_js/conftest.py

```diff
@@ -0,0 +1,58 @@
+"""Provide common Z-Wave JS fixtures."""
+import json
+from unittest.mock import patch
+
+import pytest
+from zwave_js_server.model.driver import Driver
+from zwave_js_server.model.node import Node
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+@pytest.fixture(name="controller_state", scope="session")
+def controller_state_fixture():
+    """Load the controller state fixture data."""
+    return json.loads(load_fixture("zwave_js/controller_state.json"))
+
+
+@pytest.fixture(name="multisensor_6_state", scope="session")
+def multisensor_6_state_fixture():
+    """Load the multisensor 6 node state fixture data."""
+    return json.loads(load_fixture("zwave_js/multisensor_6_state.json"))
```

Comment:
Just a note that this won't work for switches, as for switches we will merge in the value changed events when we process them.

Reply:
Actually same issue happens for sensors .